sdk/aqueduct/artifacts/table_artifact.py

```diff
@@ -174,6 +180,12 @@ def save(self, config: SaveConfig) -> None:
                 found.
         """
         integration_info = config.integration_info
+        if integration_info.service == ServiceType.ATHENA:
```

Comment:
nit: this error handling should probably belong to the `config` method that creates a `SaveConfig` for the integration, since users shouldnt be allowed to create that in the first place